modules/mobile/spec/support/helpers/spec_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.configure do |config|
+  config.before :each, type: :request do
+    Flipper.enable('va_online_scheduling')
+  end
+
+  config.define_derived_metadata(file_path: %r{modules/mobile/spec}) do |metadata|
+    metadata[:json_api] = true
```

Comment:
without this the after hook below would run for all non mobile specs as well.

Reply:
yes, it will add that metadata variable to non request items as well. I changed the name to `mobile_spec` to make the naming more inclusive.